### PR TITLE
Fix curl error on subdomain in site-command

### PR DESCRIPTION
**Pull request by sagarnasit**
Issue ref: https://github.com/EasyEngine/site-command/issues/125

### CODE REVIEW

File: src/helper/class-ee-site.php [hunk 1]
```diff
@@ -514,21 +516,48 @@ protected function init_le( $site_url, $site_fs_path, $wildcard = false ) {
...
 	}
 
+	/**
+	 * Check www is working with site domain.
+	 *
+	 * @return bool
+	 */
+	protected function check_www_subdomain():bool {
+		$random_string = random_password();
+		file_put_contents( $this->site_data['site_fs_path'] . '/app/src/ssl_check.php', $random_string );
```
Comment (mrrobot47) [inline src/helper/class-ee-site.php]: Any specific reason to keep this as `.php` file? If a random string is only going in better make it `.html` file. Also, by keeping it `.php` file it won't work for `site-type-html`.

File: src/helper/class-ee-site.php [hunk 2]
```diff
@@ -514,21 +516,48 @@ protected function init_le( $site_url, $site_fs_path, $wildcard = false ) {
...
+
+		$site_url = 'www.' . $this->site_data['site_url'] . '/ssl_check.php';
+		$curl     = curl_init();
+		curl_setopt( $curl, CURLOPT_URL, $site_url );
+		curl_setopt( $curl, CURLOPT_RETURNTRANSFER, true );
+		curl_setopt( $curl, CURLOPT_HEADER, false );
+		$data = curl_exec( $curl );
+		curl_close( $curl );
```
Comment (mrrobot47) [inline src/helper/class-ee-site.php]: Add cleanup of the created file as soon as the `curl` request is done and we have the results.

File: src/helper/class-ee-site.php (hunk 1 above)
Comment (sagarnasit) [reply src/helper/class-ee-site.php]: Making it `.txt` for convenience.

File: features/bootstrap/FeatureContext.php [hunk 3]
```diff
@@ -103,7 +101,7 @@ public function isInstalled($command)
 	}
 
 	/**
-	 * @When I run :command
+	 * @When /I run '(.*)'|"(.*)"/
```
Comment (mbtamuli) [inline features/bootstrap/FeatureContext.php]: I think this whole regex needs to be wrapped in parenthesis and the inner parenthesis are not required.
```
/I run ('.*'|".*")/
```

Comment (mrrobot47) [reply features/bootstrap/FeatureContext.php]: This regex does not work as it also matches the outer quotes.

Comment (mbtamuli) [reply features/bootstrap/FeatureContext.php]: Let's go with the one already added.
```
@When /I run '(.*)'|"(.*)"/
```

**Pull request merged**